### ISSUE TITLE
Ethereum chain mockups

### ISSUE THREAD

**Issue by acravenho** [poanetwork/poa-explorer#444]
Provide mockups for the following Ethereum networks:
- [ ] Ethereum Mainnet
- [ ] Kovan Testnet
- [ ] Rinkeby Testnet
- [ ] Ropsten Testnet

**Comment by kggp1995** [comment]
Can we connect the POA Explorer by specifying the RPC ADDR & RPC PORT of a private ethereum node?

**Comment by acravenho** [comment]
@kggp1995 Yes, please define your rpc endpoint in `apps/indexer/config/dev/parity.exs`. 

After making this change please go through the setup instructions: https://github.com/poanetwork/poa-explorer#setup-instructions

**Comment by katibest** [comment]
@acravenho Ethereum Mainnet mock up:

![etherum-homepage-03](https://user-images.githubusercontent.com/20226535/43544076-e2d77600-959f-11e8-951d-c072d5ffd356.png)

**Comment by katibest** [comment]
@acravenho OK! These should have all the updates... 

![etherum-homepage-03](https://user-images.githubusercontent.com/20226535/43544836-28f7a694-95a2-11e8-9f48-d1e9de1c3f8d.png)

![etherum-address_details-07](https://user-images.githubusercontent.com/20226535/43544857-3a83985a-95a2-11e8-9699-d66402a94f33.png)

**Comment by katibest** [comment]
@acravenho  How is this coming along? If you think this works I'll get the address details page done too!

![kovan-homepage-04](https://user-images.githubusercontent.com/20226535/43606344-0d456a04-9669-11e8-9b95-e59c38b27f75.png)

**Comment by katibest** [comment]
![kovan-address_details-08](https://user-images.githubusercontent.com/20226535/43608050-2a895c4c-966e-11e8-837e-35b24becf909.png)

**Comment by arthurra** [comment]
![rinkeby-address-details](https://user-images.githubusercontent.com/1641169/43649197-8b2caabc-970a-11e8-8004-6e35828f801b.png)
![rinkeby-homepage](https://user-images.githubusercontent.com/1641169/43649198-8b4d6e5a-970a-11e8-99e5-ffbd35307fad.png)

**Comment by pashagonchar** [comment]
Sokol test network logo
![logo_sokol](https://user-images.githubusercontent.com/16716926/43657250-e8de1db4-975d-11e8-900a-419ab706b1f0.png)
https://www.dropbox.com/s/4zcbkk9oeu9lw99/logo_sokol.psd?dl=0

**Comment by igorbarinov** [comment]
@acravenho  why there is no Sokol in the list?

**Comment by acravenho** [comment]
@igorbarinov Updated

**Comment by arthurra** [comment]
![sokol-homepage](https://user-images.githubusercontent.com/1641169/43731955-26481a48-997e-11e8-8e31-d4e13a6071ef.png)
![sokol-address-page](https://user-images.githubusercontent.com/1641169/43731961-2b2e7994-997e-11e8-95e7-ed945f0b1e0d.png)

**Comment by arthurra** [comment]
![ropsten-homepage](https://user-images.githubusercontent.com/1641169/43737211-8b902fca-998e-11e8-883f-20d36917ec07.png)
![ropsten-address-page](https://user-images.githubusercontent.com/1641169/43737214-8fcfa39a-998e-11e8-939f-5e2559fc936c.png)

**Comment by katibest** [comment]
DevCon month version (for October)!

![etherum-address_details-oct-09](https://user-images.githubusercontent.com/20226535/44109733-bf9c0da0-9fcb-11e8-9151-a83ee23ba377.png)

![etherum-homepage-oct-03](https://user-images.githubusercontent.com/20226535/44109741-c57bf140-9fcb-11e8-9841-ce9174bf9494.png)

**Comment by pashagonchar** [comment]
@katibest Why you use different font for menu?

**Comment by pashagonchar** [comment]
<img width="593" alt="graph" src="https://user-images.githubusercontent.com/16716926/44115619-0e0abb30-a017-11e8-9d19-2f169e23b47c.png">
@katibest Change text color for charts to same with color palette or maybe black

**Comment by pashagonchar** [comment]
<img width="493" alt="ropsten" src="https://user-images.githubusercontent.com/16716926/44116512-67f38e9a-a019-11e8-9da0-a15c20e764d9.png">
Same links with different colors. Design for Ropsten

**Comment by pashagonchar** [comment]
<img width="1243" alt="transaction_view" src="https://user-images.githubusercontent.com/16716926/44116778-1d5792a4-a01a-11e8-9eb6-e586709f21fa.png">
@katibest A lot of space between address and arrow. Design for Ethereum

**Comment by pashagonchar** [comment]
<img width="1243" alt="transaction_view" src="https://user-images.githubusercontent.com/16716926/44117009-bb9960c8-a01a-11e8-9029-bbbbc9a7ce7f.png">
@katibest Can you replace "->" with arrow icon. Design for Ethereum

**Comment by pashagonchar** [comment]
![ethereum_theme](https://user-images.githubusercontent.com/16716926/44118613-691e6e2e-a01f-11e8-8268-2679da401351.png)
@katibest small report for ethereum design

**Comment by arthurra** [comment]
@acravenho Can we close this issue?
